vscode/package.json

```diff
@@ -467,13 +467,33 @@
 					"command": "jdk.java.goto.super.implementation",
 					"when": "nbJdkReady && editorLangId == java && editorTextFocus",
 					"group": "navigation@100"
+				},
+				{
+					"command": "jdk.project.run",
+					"when": "nbJdkReady && editorLangId == java && resourceExtname == .java",
```

Comment:
Is there a reason to use both `editorLangId == java` and `resourceExtname == .java`? I would assume one should be enough, but don't mind too much.

Reply:
Yes only `resourceExtname == .java` would be fine I guess but to be completely confident I have added both flags to check.

Reply:
Ok. I don't object to this.